codegen/method.go

```diff
@@ -54,8 +55,9 @@ type HeaderFieldInfo struct {
 
 // MethodSpec specifies all needed parts to generate code for a method in service.
 type MethodSpec struct {
-	Name       string
-	HTTPMethod string
+	Name        string
+	MethodNames []string
```

Comment:
what are method names here ?

Reply:
this is for dedup the already ran methods - 
```
func (ms *MethodSpec) isMethodprinted(methodName string) bool {
	sort.Strings(ms.MethodNames)
	i := sort.SearchStrings(ms.MethodNames, methodName)
	if i < len(ms.MethodNames) && ms.MethodNames[i] == methodName {
		return true
	}
	return false
}
```

Reply:
methodnames is confusing because its on the MethodSpec.

Maybe we want `ConverterFunctionNames`